src/oop/special_class_method/hard/q5/QuestionText.md

```diff
@@ -0,0 +1,41 @@
+# oop-special_class_method-hard-q5
+
+### Question Text
+
+You are writing a Java program that describes different types of roads and their attributes. Write a Java class called
+Road the has the following description:
```

Comment:
You should also indicate the student will need to write a test class or else they might miss it by mistake.